docs/synchronization2_layer.md

```diff
@@ -138,3 +133,6 @@ Which you can set in the following way:
 
 <br></br>
 
+## Layer Options
+
+The options for this layer are specified in VkLayer_khronos_synchronization2.json. The option details are in [synchronization2_layer.html](https://vulkan.lunarg.com/doc/sdk/latest/windows/synchronization2_layer.html).
```

Comment:
Change "The option details are in [synchronization2_layer.html]" to

"The details of the layer options is documented in the [Synchronization2 layer documentation]"

Reply:
I'll make this change, but shouldn't it say "The details of the layer options are documented in ..."